README.md

```diff
@@ -57,4 +57,6 @@ correct version of `pycmake` into your *libecl* code:
     git submodule update --init pycmake
 
 
-
+### Installing the Python code ###
+    * pip install .
+    * check with: python -c "import ert.ecl as ecl"
```

Comment:
The `ert` package is quite deprecated - better to use e.g check with: `python -c import ecl`

Reply:
done

> > Unfortunately my time is quite pressed - so if you have a fully working Travis + PyPI pipeline I would be very grateful if you could add a commit illustrating that - otherwisse I will eventually figure it out.
> 
> OK - I have looked briefly on the Travis + PyPi integration, that looks quite managable; you don't need to worry about that.

thanks. I don't actually know this so very curious what this actually means.